ObsoleteFiles.inc

```diff
@@ -51,6 +51,9 @@
 #   xargs -n1 | sort | uniq -d;
 # done
 
+# 20241102: remove the outdated c2x.7 in favor of c23.7
```

Comment:
I would be explicit that this is the c2x.7 symlink, not a separate page

Reply:
Alright, will do it now.

Reply:
Done.